docs/guide/controller-plugin.md

```diff
@@ -68,11 +68,21 @@ Robot windows are implemented in HTML and provide the following features:
 The equivalent controller functions are `wb_robot_wwi_receive_text` and `wb_robot_wwi_send_text`.
 3. They are web-ready and could be used to display robot windows on web pages.
 
-A simple example of an HTML robot window is provided in the `robots/thymio/thymio2.wbt` sample simulation and demonstrates:
-
-- How to display sensor information in the robot window.
-- How to send user interface events (like mouse clicks) from the robot window to the controller program.
-- How to change the title of the robot window from the controller program.
+A simple example of an HTML robot window is given in [`projects/samples/howto/custom_robot_window_simple`](samples-howto.md#custom_robot_window_simple-wbt) and it demonstrates how to establish a two-way communication between a robot window and Python controller.
+To create a similar robot window for your project follow these steps:
+1. In your project's root create a file in the following path `plugins/robot_windows/<robot window name>/<robot window name>.html`.
+2. The file is a typical HTML document that can contain JavaScript and CSS.
+However, in addition to the standard JavaScript library, a `webots` object is injected that exposes an interface to allow communication with a robot controller.
+Therefore, you can check usage examples of `webots.window("<robot window name>").receive` and `webots.window("<robot window name>").send` in [`projects/samples/howto/custom_robot_window_simple/plugins/robot_windows/custom_robot_window_simple/custom_robot_window_simple.html`](https://github.com/cyberbotics/webots/tree/master/projects/samples/howto/custom_robot_window_simple/plugins/robot_windows/custom_robot_window_simple/custom_robot_window_simple.html).
```

Comment:
This is correct in this PR.
But note that since the introduction of the `released` branch we have now issues with this kind of links because we need to check and update (if new have been added) every time we release a new version.

Reply:
But this problem is not specific to this PR, it is project-wide, this is not the only occurrence of a GitHub link. Maybe, we can improve the documentation engine to handle branches automatically? Like if we write:
```md
[`path/to/file`](https://github.com/cyberbotics/webots/tree/{version}/path/to/file)
```
it should resolve `{version}` placeholder as a branch name.

Reply:
Yes, it is not specific to this PR but this is the first time we are re-introducing this issue.
I just wanted to point out the problem because after merging this we should fix it again.
We should open an issue and fix this as soon as possible.

Reply:
Yes, it is not specific to this PR. But this is the first time we introduce again a link that we will need to fix later.
I just wanted to point out the problem and warn that this will need to be fixed after merging it.
It would be good to open an issue about this and fix it as soon as possible.